### PR TITLE
Add firebase observability to the playground

### PR DESCRIPTION
**Pull request by paweljanicki**
#### After

### CODE REVIEW

File: src/services/FirestoreService.ts [hunk 1]
```diff
@@ -240,10 +241,24 @@ export class FirestoreService<
...
-  public subscribeToAllPublicDeals(): Observable<Array<IFirebaseDocument>> {
+  public allPublicDealsObservable(): Observable<Array<IFirebaseDocument>> {
     return this.getObservableOfQuery<Array<IFirebaseDocument>>(this.allPublicDealsQuery());
   }
 
+  public updatesToAllPublicDeals(): Observable<{
+    added: Array<IDealTokenSwapDocument>,
+    modified: Array<IDealTokenSwapDocument>,
+    removed: Array<IDealTokenSwapDocument>,
+  }> {
```
Comment (paweljanicki) [inline src/services/FirestoreService.ts:252]: This is a method where we listen to what was added, modified and removed @dkent600

**Comment by dkent600** [comment]
Ooops, sorry, didn't notice this was just a draft.  So I won't merge.

**Review by dkent600 — APPROVED** [review]

**Review by dkent600 — CHANGES_REQUESTED** [review]
To the extend this is about introducing rxjs, then I wouldn't want to merge it.

**Comment by paweljanicki** [comment]
Not intended to be merged

**Pull request closed without merging**